packages/opentelemetry-core/test/metrics/NoopMeter.test.ts

```diff
@@ -0,0 +1,71 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as assert from 'assert';
+import { DistributedContext, SpanContext } from '@opentelemetry/types';
+import {
+  NoopMeter,
+  NOOP_GAUGE_HANDLE,
+  NOOP_GAUGE_METRIC,
+  NOOP_COUNTER_HANDLE,
+  NOOP_COUNTER_METRIC,
+  NOOP_MEASURE_HANDLE,
+  NOOP_MEASURE_METRIC,
+} from '../../src/metrics/NoopMeter';
+
+describe('NoopTracer', () => {
+  it('should not crash', () => {
+    const meter = new NoopMeter();
+    const counter = meter.createCounter('some-name');
+    // ensure NoopMetric does not crash.
+    counter.setCallback(() => {
+      assert.fail('callback occurred');
+    });
+    counter.getHandle(['val1', 'val2']).add(1);
+    counter.getDefaultHandle().add(1);
+    counter.removeHandle(['val1', 'val2']);
+
+    // ensure the correct noop const is returned
+    assert.strictEqual(counter, NOOP_COUNTER_METRIC);
+    assert.strictEqual(
+      counter.getHandle(['val1', 'val2']),
+      NOOP_COUNTER_HANDLE
+    );
+    assert.strictEqual(counter.getDefaultHandle(), NOOP_COUNTER_HANDLE);
+    counter.clear();
+
+    const measure = meter.createMeasure('some-name');
+    measure.getDefaultHandle().record(1);
+    measure.getDefaultHandle().record(1, {} as DistributedContext);
+    measure.getDefaultHandle().record(1, {} as DistributedContext, {} as SpanContext);
```

Comment:
Instead of using type assertion, I'd suggest to pass actual `DistributedContext` and `SpanContext` object.
Something like:
```
measure.getDefaultHandle().record(1, {'key': {value: 'value'});
measure.getDefaultHandle().record(1, {'key': {value: 'value'}, 
    {traceId: 'a3cda95b652f4a1592b449d5929fda1b', spanId: '5e0c63257de34c92'});
```

WDYT?